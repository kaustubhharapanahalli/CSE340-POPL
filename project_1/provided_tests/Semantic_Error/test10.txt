t1b (((t)|(y))*).((t).(y)),
t1b (((t)|(y))*).((f).(m)),
t1b (((t)|(y))*).((t).(g)) #
" tytyyttt yyttyt ty "
