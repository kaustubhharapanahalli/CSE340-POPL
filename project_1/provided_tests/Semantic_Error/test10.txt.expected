Line 2: t1b already declared on line 1
Line 3: t1b already declared on line 1
